robust/robust.py

```diff
@@ -7,12 +9,12 @@
 import warnings
 from scipy.stats import norm
 
-from robust_gp_tools import RobustGPTools
-from equivalent_posynomials import EquivalentPosynomials
-from equivalent_models import TwoTermBoydModel
-from twoterm_approximation import TwoTermApproximation
-from robustify_large_posynomial import RobustifyLargePosynomial
-from linearize_twoterm_posynomials import LinearizeTwoTermPosynomials
+from .robust_gp_tools import RobustGPTools
+from .equivalent_posynomials import EquivalentPosynomials
+from .equivalent_models import TwoTermBoydModel
+from .twoterm_approximation import TwoTermApproximation
+from .robustify_large_posynomial import RobustifyLargePosynomial
+from .linearize_twoterm_posynomials import LinearizeTwoTermPosynomials
```

Comment:
Why do we suddenly need all these absolute imports? Seems strange? I've sometimes seen issues with this if I don't install the package before running tests, but if you `python setup.py sdist` and `pip install .` are the absolute imports really needed?

Reply:
I personally really like the relative imports here, because the directory structure is unlikely to change and it makes it clear where the actual methods/files are. It's a relatively minor change and should not affect functionality at all.